docs/reference/topology-updater-commandline-reference.md

```diff
@@ -169,17 +169,31 @@ Example:
 nfd-topology-updater -watch-namespace=rte
 ```
 
-### -kubelet-config-file
+### -kubelet-config-uri
 
-The `-kubelet-config-file` specifies the path to the Kubelet's configuration
-file.
+The `-kubelet-config-uri` specifies the path to the Kubelet's configuration.
+Note that the URi could either be a local host file or an HTTP endpoint.
 
-Default:  /host-var/lib/kubelet/config.yaml
+Default:  `https://${NODE_NAME}:10250/configz`
 
 Example:
 
 ```bash
-nfd-topology-updater -kubelet-config-file=/var/lib/kubelet/config.yaml
+nfd-topology-updater -kubelet-config-uri=file:///var/lib/kubelet/config.yaml
+```
+
+### -api-auth-token-file
+
+The `-api-auth-token-file` specifies the path to the token file which is used to
+retrieve Kubelet's configuration from Kubelet secure port, if not specified,
+default to `/var/run/secrets/kubernetes.io/serviceaccount/token`.
+Note that this token file must bind to a role that has the `get` capability to
+`nodes/proxy` resources.
```

Comment:
nit: maybe use the same structure as in the other flags and mention default separately.

```suggestion
retrieve Kubelet's configuration from Kubelet secure port.
Note that this token file must bind to a role that has the `get` capability to
`nodes/proxy` resources.

Default: `/var/run/secrets/kubernetes.io/serviceaccount/token`
```

Also,  might want to note that this flag is only used when http endpoint is specified in the `- kubelet-config-uri`